aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -229,10 +218,10 @@ public void testRenameTable() {
     glueCatalog.renameTable(
         TableIdentifier.of(namespace, tableName), TableIdentifier.of(namespace, newTableName));
     Table renamedTable = glueCatalog.loadTable(TableIdentifier.of(namespace, newTableName));
-    Assert.assertEquals(table.location(), renamedTable.location());
-    Assert.assertEquals(table.schema().toString(), renamedTable.schema().toString());
-    Assert.assertEquals(table.spec(), renamedTable.spec());
-    Assert.assertEquals(table.currentSnapshot(), renamedTable.currentSnapshot());
+    assertThat(renamedTable.location()).isEqualTo(table.location());
+    assertThat(renamedTable.schema()).asString().isEqualTo(table.schema().toString());
```

Comment:
nit: Just wondering why cast to `asString()` here ? Other Object comparison we are doing using `isEqualTo`

Reply:
It's necessary because the testing the comparisons between objects fail.

Reply:
Yeah Just wondering , Why PartiotionSpec and Snapshot comparison are working. 
As per my understanding `isEqualTo` compares object values. So ideally it should not fail  unless I am missing something.

Reply:
it's because `Schema` doesn't implement equals()